java-base/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+LABEL maintainer="Stackable GmbH"
+
+# https://github.com/hadolint/hadolint/wiki/DL4006
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ARG PRODUCT
+ARG _SECURITY_PATH
+
+RUN microdnf update && \
+    microdnf install java-${PRODUCT}-openjdk-headless --nodocs && \
```

Comment:
Just wanted to make sure if we want to install a JRE or a JDK?
From the Image size and security perspective i would think a JRE like `java-${PRODUCT}-openjdk-jre-headless` should be sufficient

Reply:
From the daily JDK is fine for now as some products need a JDK